### ISSUE TITLE
App is not working with uber &left

### ISSUE THREAD

**Issue by Malsaab** [adriangl/pict2cam#214]
Hello brother'
Your app work very good  .but The big fat issue is . Its not working with some apps at all like uber driver and left and bolt driver  these apps direct open the camera and seem to ignore giving me a choice  . 
Do u have any idea what might be the issue  ?

**Comment by grigotierpascal** [comment]
Same issue, some app block it. Have you tried with android 10 ?

**Comment by Malsaab** [comment]
> Same issue, some app block it. Have you tried with android 10 ?

Yes i have s9 pluse with android 10 . I brougth it just for this  xd

**Comment by marmarisweb** [comment]
Is the problem solved? I couldn't do it either. What can I do for Uber?
